src/include/optimizer/stats/cost.h

```diff
@@ -24,30 +24,39 @@
 namespace peloton {
 namespace optimizer {
 
-// Default cost when cost model cannot compute correct cost.
-static constexpr double DEFAULT_COST = 1;
 
-// Estimate the cost of processing each row during a query.
-static constexpr double DEFAULT_TUPLE_COST = 0.01;
+//===----------------------------------------------------------------------===//
+// Cost
+//===----------------------------------------------------------------------===//
+class Cost {
+ private:
+	// Default cost when cost model cannot compute correct cost.
+	static constexpr double DEFAULT_COST = 1;
```

Comment:
I don't think those constant definitions can be moved inside of the Cost class. This will cause compile errors.